planning/obstacle_cruise_planner/src/optimization_based_planner/velocity_optimizer.cpp

```diff
@@ -252,25 +252,29 @@ VelocityOptimizer::OptimizationResult VelocityOptimizer::optimize(const Optimiza
     std::cerr << "optimization failed : " << qp_solver_.getStatusMessage().c_str() << std::endl;
   }
```

Comment:
Is there are reason not to return early here?

Reply:
Good point. I updated the logic to also skip filling the `optimized_result` in that case.